glassfish-tck-runner/pom.template.xml

```diff
@@ -16,7 +16,7 @@
     SPDX-License-Identifier: EPL-2.0 OR GPL-2.0 WITH Classpath-exception-2.0
 
 -->
-
+<!-- Portions Copyright [2022] [Payara Foundation and/or its affiliates] -->
```

Comment:
Payara is already above in the original blob, it just needs the year updating.
"Portions Copyright" is something specific to the dual license Payara uses I believe, not EPL.

Reply:
already done